carma_cooperative_perception/CMakeLists.txt

```diff
@@ -133,8 +133,10 @@ if(carma_cooperative_perception_BUILD_TESTS)
   )
 
   add_launch_test(test/track_list_to_external_object_list_launch_test.py)
-  add_launch_test(test/multiple_object_tracker_duplicates_launch_test.py)
-  add_launch_test(test/multiple_object_tracker_static_obstacle_launch_test.py)
+  # This test has been temporarily disabled to support Continuous Improvement (CI) processes.
+  # Related GitHub Issue: <https://github.com/usdot-fhwa-stol/carma-platform/issues/2335>
+  # add_launch_test(test/multiple_object_tracker_duplicates_launch_test.py)
+  # add_launch_test(test/multiple_object_tracker_static_obstacle_launch_test.py)
```

Comment:
@MishkaMN @adamlm do we want to prioritize fixing these unit tests as part of the release for VRU?